website/source/assets/stylesheets/application.scss

```diff
@@ -22,6 +22,8 @@
 @import '_header';
 @import '_footer';
 @import '_buttons';
+@import '_sidebar';
```

Comment:
@briankassouf There is a `website/source/layouts/_sidebar.erb` file, which I would guess is that one.